docs/integrations/glue_metastore.md

```diff
@@ -0,0 +1,394 @@
+
+### What is supported 
+
+- Creating a unique table in Glue Catalog per lakeFS repository / ref / commit. 
+- No data copying is required, the table location is a path to a symlinks structure in S3 based on Hive's [SymlinkTextInputFormat](https://svn.apache.org/repos/infra/websites/production/hive/content/javadocs/r2.1.1/api/org/apache/hadoop/hive/ql/io/SymlinkTextInputFormat.html) and the [table partitions](https://docs.aws.amazon.com/glue/latest/dg/tables-described.html#tables-partition) are maintained.
+- Tables are described via [Hive format in `_lakefs_tables/<my_table>.yaml`]({% link howto/catalog_exports.md %}#hive-tables).
+- Currently the data query in Glue metastore is Read-Only operation and mutating data requires writting to lakeFS and letting the export hook run.
+
+### How it works 
+
+Based on event lakeFS events such as `post-commit` an Action will run a script that will create Symlink structures in S3 and then will register a table in Glue.
+The Table data location will be the generated Symlinks root path.
+
+There are 4 key pieces:
+
+1. Table description at `_lakefs_tables/<your-table>.yaml`
+2. Lua script that will do the export using [symlink_exporter]({% link howto/hooks/lua.md %}#lakefscatalogexportsymlink_exporter) and [glue_exporter]({% link howto/hooks/lua.md %}#lakefscatalogexportglue_exporter) packages.
+3. [Action Lua Hook]({% link howto/catalog_exports.md %}#running-an-exporter) to execute the lua hook.
+4. Write some lakeFS table data ([Spark]({% link integrations/spark.md %}), CSV, etc)
+
+To learn more check [Data Catalog Exports]({% link howto/catalog_exports.md %}).
+
+## Example: Using Athena to query lakeFS data
+
+### Pre-requisite 
+
+1. Glue Database to use (lakeFS does not create a database).
+2. AWS Credentials with permission to manage Glue, Athena Query and S3 access.
+3. lakeFS [Actions]({% link howto/hooks/index.md %}) enabled and configured with S3 blockstore.
+
+### Create lakeFS repository 
+
+Let's create a repository named `catalogs`. 
+
+```bash
+lakectl repo create lakefs://catalogs s3://<lakefs-bucket>/<repo-prefix>
+```
+### Add table descriptor
+
+Let's define a table and commit to lakeFS. 
+Save the YAML below as `animals.yaml` and upload it to lakeFS. 
+
+```bash
+lakectl fs upload lakefs://catalogs/main/_lakefs_tables/animals.yaml -s ./animals.yaml && \
+lakectl commit lakefs://catalogs/main -m "added table"
+```
+
+```yaml 
+name: animals
+type: hive
+# data location root in lakeFS
+path: tables/animals
+# partitions order
+partition_columns: ['type', 'weight']
+schema:
+  type: struct
+  # all the columns spec
+  fields:
+    - name: type
+      type: string
+      nullable: true
+      metadata:
+        comment: axolotl, cat, dog, fish etc
+    - name: weight
+      type: integer
+      nullable: false
+      metadata: {}
+    - name: name
+      type: string
+      nullable: false
+      metadata: {}
+```
+
+### Write some table data
+
+Add some table data. 
```

Comment:
I suggest to keep this section short, even a one-liner if that's possible. It's not in the scope of the article, it's more like a means to an end.
It can be something like "Insert data under the table path, using your preferred method. For example, here is how to write data using Spark: ..."

The commit bit doesn't need to be in the code, you can simply say "then perform a commit in lakeFS".

Reply:
@itaiad200 @arielshaqed opinions?

Reply:
@johnnyaug is right that this doesn't belong in "integrations".  Unfortunately if you remove this text, it means you've just been volunteered to write the blog post / tech note.

Reply:
@arielshaqed so setting aside volunteering, you agree that this step by step tutorial on how to write the data itself should be removed to a general guidance? :)

Reply:
@johnnyaug I took your advice and removed 99% of the content from there!